examples/templateservicebroker/templateservicebroker-template.yaml

```diff
@@ -138,3 +139,57 @@ objects:
   - kind: ServiceAccount
     namespace: ${NAMESPACE}
     name: apiserver
+
+# for the SA for the service catalog to use when calling the tsb
+- apiVersion: v1
+  kind: ServiceAccount
+  metadata:
+    namespace: ${NAMESPACE}
+    name: templateservicebroker-client
+
+# allow the SC's SA to call the TSB
+- kind: ClusterRoleBinding
+  apiVersion: v1
+  metadata:
+    name: templateservicebroker-client
+  roleRef:
+    name: system:openshift:templateservicebroker-client
+  userNames:
+  - system:serviceaccount:${NAMESPACE}:templateservicebroker-client
+
+# copy of the SC's SA's token secret which will be retrieved by the SC for use 
+# when calling the TSB
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    namespace: ${NAMESPACE}
+    name: templateservicebroker-client
+    annotations:
+      kubernetes.io/service-account.name: templateservicebroker-client
+  type: kubernetes.io/service-account-token
+
+# allow the SC to read the SA's secret defined above
+- kind: Role
+  apiVersion: v1
+  metadata:
+    name: templateservicebroker-auth-reader
+    namespace: ${NAMESPACE}
+  rules:
+  - apiGroups:
+    - ""
+    resourceNames:
+    - templateservicebroker-client
+    resources:
+    - secrets
+    verbs:
+    - get
+- kind: RoleBinding
+  apiVersion: v1
+  metadata:
+    namespace: ${NAMESPACE}
+    name: templateservicebroker-auth-reader
+  roleRef:
+    namespace: ${NAMESPACE}
```

Comment:
This shouldn't be needed (no similar field in rbac)

Reply:
the role is required to exist in this namespace?

Reply:
the field certainly exists (though it may not be needed here), or how is this working?
https://github.com/openshift/origin/blob/master/examples/templateservicebroker/templateservicebroker-template.yaml#L135

Reply:
> the role is required to exist in this namespace?

In RBAC, you cannot specify a namespace.  In the openshift authorization objects, empty means, "use current".